services/job_runner_test.go

```diff
@@ -149,13 +149,14 @@ func TestJobRunner_ExecuteRun(t *testing.T) {
 		runResult  string
 		wantStatus models.RunStatus
 		wantData   string
+		wantBody   string
 	}{
-		{"success", bridgeName, `{}`, `{"data":{"value":"100"}}`, models.RunStatusCompleted, `{"value":"100","type":"auctionBidding"}`},
-		{"errored", bridgeName, `{}`, `{"error":"too much"}`, models.RunStatusErrored, `{"type":"auctionBidding"}`},
-		{"errored with a value", bridgeName, `{}`, `{"error":"too much", "data":{"value":"99"}}`, models.RunStatusErrored, `{"value":"99","type":"auctionBidding"}`},
-		{"overriding bridge type params", bridgeName, `{"url":"hack"}`, `{"data":{"value":"100"}}`, models.RunStatusCompleted, `{"value":"100","url":"hack","type":"auctionBidding"}`},
-		{"type parameter does not override", bridgeName, `{"type":"0"}`, `{"data":{"value":"100"}}`, models.RunStatusCompleted, `{"value":"100","type":"0"}`},
-		{"non-existent bridge type", "non-existent", `{}`, `{}`, models.RunStatusErrored, `{}`},
+		{"success", bridgeName, `{}`, `{"data":{"value":"100"}}`, models.RunStatusCompleted, `{"value":"100","type":"auctionBidding"}`, `{"type":"auctionBidding"}`},
```

Comment:
@se3000/@thodges-gh  should wantBody contain `"value":"100"` ?

Reply:
Yes, and `"type":"auctionBidding"`.

Reply:
Cool, looks like the tests weren't asserting for this and it was confusing us. Will fix in this PR.

Reply:
Hmm, circling back on this - `"value":"100"` is contained in the response body of the bridgeAdapter, so it cannot be submitted to the bridgeAdapter (without violating causality).

I believe these tests are all fine, I've just changed them to be more explicit.